common/artifacts/909_wc_artifacts_library_librams.txt

```diff
@@ -14,7 +14,8 @@ libram_of_protection = {
 	health = 0.9
 	combat_rating = 12
 
-	monthly_character_piety = 0.5
+	monthly_character_piety = 0.45
+	monthly_character_prestige = 0.2
```

Comment:
can you make it 0.4/0.2 or 0.45/0.1 ?

Reply:
and it's already again 1.4 stronger then other legendary artifacts

Reply:
they are not 1.4 times stronger though

Reply:
i cant make it 0.4/0.2 cause the math wont work

Reply:
not 1.4 times, but 26.4 cost instead of the usually 25 or lower

Reply:
it can either linear or 25